ale_linters/c/flawfinder.vim

```diff
@@ -4,6 +4,7 @@
 call ale#Set('c_flawfinder_executable', 'flawfinder')
 call ale#Set('c_flawfinder_options', '')
 call ale#Set('c_flawfinder_minlevel', 1)
+call ale#Set('flawfinder_error_severity', 6)
```

Comment:
This should be named `c_flawfinder_error_severity`.

Reply:
I was unsure about this one.  This variable affects both C and C++.